.github/workflows/ci.yaml

```diff
@@ -114,25 +119,30 @@ jobs:
       matrix:
         k8s:
           - v1.18.20
-          - v1.22.9
-          - v1.24.2
+          - v1.22.17
+          - v1.24.17
           - v1.25.16
-          - v1.26.13
+          - v1.26.15
+          - v1.27.16
+          - v1.28.14
+          - v1.29.9
+          - v1.30.5
```

Comment:
same for 1.35.5

```
➜ crane ls kindest/node | grep 1.30
v1.30.0
v1.30.2
v1.30.3
v1.30.4
```

Reply:
Interesting. The schemas have been present for a bit. 
https://github.com/yannh/kubernetes-json-schema/tree/master/v1.29.9
https://github.com/yannh/kubernetes-json-schema/tree/master/v1.30.5

so maybe they just haven't built those as Docker images yet...

I can just pin all the versions to the same set of versions across both vs. doing new ones for the schemas and older ones for the docker tests?